src/data/users.js

```diff
@@ -1,6 +1,6 @@
 // Do not edit this file
 
-const userData = [
+module.exports = [
   {
     "id": 1,
     "name": "Luisa Hane",
```

Comment:
The only way that I was able to view the users.js data on the console (in Chrome) was by exporting it so that our scripts.js file could use it on line 14 via import. I believe this is what is needed in order to be able to manipulate the data for our dashboard prior to Iteration 2.